overthebox/defaults/network.defaults

```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
 EOF
 fi
 
+uci get network.globals.tcp_congestion_control >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+  uci -q batch <<-EOF >/dev/null
+	set network.globals=globals
+	set network.globals.tcp_congestion_control='cubic'
```

Comment:
nothing about this file ? https://github.com/ovh/overthebox-feeds/blob/master/overthebox/defaults/tcp_congestion_control.defaults